### PR TITLE
Python script

### PR DESCRIPTION
**Pull request by kapkic**
3 scripts: Load_python, load_bash, test_python

### CODE REVIEW

**Comment by rennergade** [comment]
These look good! I think mostly a question of in the long term where we want to store some of this stuff. @jesings should we create like a loadable dummy FS for some of these things that we want to add regardless? Things like `/etc/passwd`

**Comment by kapkic** [comment]
If `/etc/passwd` is a file that we'd like to always have it on lind I can also add it to mklind-install process.

File: src/scripts/load_bash.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+cd /home/lind/lind_project/tests/applications/bash/
+./bootstrap_nacl || true;
```
Comment (jesings) [inline src/scripts/load_bash.sh:2]: What's the purpose of the true here? It doesn't seem like there is anything happening with the return value

File: src/scripts/load_python.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+echo "Building python"
+cd /home/lind/lind_project/tests/applications/python/;
+./bootstrap_nacl || true;
```
Comment (jesings) [inline src/scripts/load_python.sh:3]: And again?

**Review by jesings — COMMENTED** [review]
Awesome stuff! Maybe adding in a coreutils script would be nice considering you've got the other ones down pretty well

File: src/scripts/load_bash.sh (hunk 1 above)
Comment (kapkic) [reply src/scripts/load_bash.sh:2]: These were to ensure that the script continues to run when there's an error occurs during building (inevitable errors like python has). But it seems that they does not affect our case as you've also said.
Removed.

File: src/scripts/load_python.sh (hunk 2 above)
Comment (kapkic) [reply src/scripts/load_python.sh:3]: Removed.

**Comment by kapkic** [comment]
Coreutils's a great idea! Just added it. For the time being it only copies the executables in src folder.

**Review by jesings — APPROVED** [review]
Looks really great!

**Comment by rennergade** [comment]
Cool, I think this is good for now. We'll figure out setting up filesystem later.

**Pull request merged**